semantic_map/RegionInstance semantic_pose
geometry_msgs/PoseStamped geometric_pose

